src/components/DataReportSection/index.js

```diff
@@ -90,7 +90,11 @@ const DataReportSection = () => {
   )
 
   return (
-    <div css={section}>
+    <div
+      css={section}
+      onMouseEnter={() => swiperRef.current.autoplay.stop()}
+      onMouseLeave={() => swiperRef.current.autoplay.start()}
```

Comment:
`swiperRef.current` can be null. Check for null before calling function

Reply:
fixed.